pom.xml

```diff
@@ -1,73 +1,19 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>io.horizen</groupId>
-	<artifactId>eon</artifactId>
-	<version>0.1.0-SNAPSHOT6</version>
+	<artifactId>eonproject</artifactId>
+	<version>0.1.0-SNAPSHOT7</version>
```

Comment:
For CI/CD it will be preferred not to add `digit` in the end of the version number after SNAPSHOT. Same as we do it in all our other package builds pipelines. For example SDK: https://github.com/HorizenOfficial/Sidechains-SDK/blob/dev/pom.xml